include/zephyr/bluetooth/l2cap.h

```diff
@@ -111,6 +111,20 @@ extern "C" {
  */
 #define BT_L2CAP_ECRED_MIN_MPS 64
 
+/** @brief L2CAP maximum MTU
+ *
+ *  The maximum MTU for an L2CAP Based Connection. This is the same with or without ECRED. This
+ *  requirement is taken from text in Core 3.A.4.22 and 3.A.4.26 v6.0.
+ */
+#define BT_L2CAP_MAX_MTU UINT16_MAX
```

Comment:
I have a slight preference to defining this as a literal value (i.e. 65535U or 0xFFFFU), but it's OK as is

Reply:
I'll keep it as is unless I get any objections. However, doing a lap and adding the literal type to defines would be nice at a later time!